modin/config/envvars.py

```diff
@@ -65,7 +65,7 @@ def _get_default(cls):
         except ImportError:
             pass
         else:
-            if version.parse(ray.__version__) < version.parse("1.0.0"):
+            if version.parse(ray.__version__) < version.parse("1.3.0"):
```

Comment:
```suggestion
            if version.parse(ray.__version__) <= version.parse("1.2.0"):
```

Reply:
We can't require ray version < 1.3.0 because we use environment-dev.yml, which installs 1.0.0, in CI.